CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Why does this project need a CMakeLists.txt? It is header only.

Reply:
Most Modern IDE support cmake, such as CLion、QT Creator.
So, for better notice/warning by IDE, even for simple test example, a CMakeLists.txt is useful.
Otherwise, most people will create one for test(especially when use IDE).

Finally, this CMakeLists only for test example, user can just ignore it.